projects/core/src/configurator/commons/store/reducers/configurator.reducer.ts

```diff
@@ -145,6 +145,30 @@ export function reducer(
   }
   return state;
 }
+
+function setInitialCurrentGroup(
+  state: Configurator.Configuration
+): Configurator.Configuration {
+  if (state.interactionState.currentGroup) {
+    return state;
+  }
+  let initialCurrentGroup = null;
+
+  if (state?.groups?.length > 0) {
```

Comment:
it has to be an IF statement, state?groups[0]?.id did not work when groups itself is null and lead to an error.